Focusgroup/explainer.md

```diff
@@ -207,58 +278,88 @@ Some built-in controls like `<input type=text>` "trap" nearly all keys that woul
 focusgroup. This proposal does not provide a way to prevent this from happening (Tab should continue
 to be used to "exit" these trapping elements).
 
-### 6.3. Configuring and customizing the focusgroup
-
-The focusgroup attribute can be used like a boolean attribute as described previously. However, it
-also supports various configuration and customization options that may be specified as space-separated
-tokens in the attribute's value.
-
-#### 6.3.1. 'wrap' - Wrapping focus
-
-By default, the focusgroup only provides linear traversal of the items in the group. When focus reaches
-a boundary, it stops. To enable wrap-around focus behavior, add the `wrap` token to the attribute.
-
-#### 6.3.2. 'extend' - Extending the focusgroup
-
-By default, the focusgroup's scope only covers its direct children. To extend the reach of the focusgroup,
-simply declare a second focusgroup attribute on one of the original focusgroup's direct children, and 
-indicate that this group will extend the parent's group. It becomes an extension of the original group.
-
-Below, the `<my-accordion>` element with a focusgroup attribute defines a focusgroup with nothing focusable
-in it. :( This is because the focusable `<button>` elements are separated by a level in the hierarchy:
+### 6.3. Enabling wrapping behaviors
+
+By default, focusgroup traversal with arrow keys ends at boundaries of the focus group (the start and
```

Comment:
I've noticed that sometimes you say "focusgroup" and other times "focus group"... am I oblivious to some logic here that explains the difference?

Reply:
If we were to settle on one, I'd have a preference for "focusgroup", but it's probably just 
1. I'm used to talking about "focusgroup"
2. Mentioning "focus group" makes me think about a this: https://en.wikipedia.org/wiki/Focus_group

Reply:
Did a consistency pass to replace all `/focus\s+group/` with "focusgroup".